include/tc/core/cuda/cuda_tc_executor.h

```diff
@@ -26,11 +26,6 @@
 
 namespace tc {
 
-using PpcgppCompilationCommand = std::function<std::string(
-    std::unique_ptr<tc::polyhedral::Scop>&& scop,
-    const std::vector<const DLTensor*>& inputs,
-    const MappingOptions& options)>;
```

Comment:
I missed this one.  I was only grepping for `PPCG` and `ppcg`.
Surely, this can go in independently.